pennylane/drawer/tape_mpl.py

```diff
@@ -215,14 +215,15 @@ def tape_mpl(
 
     .. code-block:: python
 
-        with qml.tape.QuantumTape() as tape:
-            qml.QFT(wires=(0,1,2,3))
-            qml.IsingXX(1.234, wires=(0,2))
-            qml.Toffoli(wires=(0,1,2))
-            qml.CSWAP(wires=(0,2,3))
-            qml.RX(1.2345, wires=0)
+        ops = [qml.QFT(wires=(0,1,2,3)),
```

Comment:
```suggestion
        ops = [
            qml.QFT(wires=(0,1,2,3)),
```
Very minor :laughing: